smurfs/src/reducers/index.js

```diff
@@ -1,7 +1,50 @@
+import { 
+  FETCH_SMURFS, 
+  SUCCESS, 
+  FAILURE, 
+  POST_SMURF, 
```

Comment:
You have POST_SMURF and DELETE_SMURF here but you are not using any of them, however you are sending the type POST_SMURF when performing the post request.